### ISSUE TITLE
【功能建议】能不能在程序名称上点击右键显示右键菜单

### ISSUE THREAD

**Issue by vr2009** [yingDev/WGestures2-mac-bugs#92]
你好，yinDev
用了两个月，随着应用的增加，导致对应用的分组越来越麻烦，把应用拖放到分组内用鼠标还是和麻烦的，而且还会打乱之前的程序排列，建议增加把应用添加到分组的右键菜单操作，这样可以一键直达，省去了拖放的操作，而且拖放不能用滚轮上滚也是很麻烦的事情
下图是我制作的效果图
![给鼠标手势的功能建议-添加邮件菜单](https://user-images.githubusercontent.com/6814896/143511070-d9b0c508-7c72-41e6-8e18-39fd81bbf516.jpg)

**Comment by superDMS** [comment]
是的  手势太多的情况下  光靠搜索来管理显得有些捉襟见肘了  ,应当加入  批量选中  多选  复制粘贴手势  等操作  方便批量添加相似的手势     类似gesturesign 那样的手势管理挺好的  希望可以在后期版本中陆续完善.

**Comment by vr2009** [comment]
> 是的 手势太多的情况下 光靠搜索来管理显得有些捉襟见肘了 ,应当加入 批量选中 多选 复制粘贴手势 等操作 方便批量添加相似的手势 类似gesturesign 那样的手势管理挺好的 希望可以在后期版本中陆续完善.

是的，现在应用多了分类不好管，我上次的鼠标手势也是gesturesign，这个鼠标是手势很适合我用的，功能很齐全，缺点就是界面我觉得有点丑了，都是上代win8的大色块风格，作者也不维护了

**Comment by superDMS** [comment]
> 

还是wg好用  gs纯手势有点落后了  加上ui不好看  哈哈  但他的手势管理可以借鉴  非常方便批量管理

**Comment by vr2009** [comment]
> > 
> 
> 还是wg好用 gs纯手势有点落后了 加上ui不好看 哈哈 但他的手势管理可以借鉴 非常方便批量管理

wg手势触发我觉得是用过最快的，没什么拖泥带水，但是一些功能性稍缺，我现在都不知道怎么粘贴大段文字，还要带标点符号那种的，你知道怎么操作吗？？

**Comment by superDMS** [comment]
> > > 
> > 
> > 
> > 还是wg好用 gs纯手势有点落后了 加上ui不好看 哈哈 但他的手势管理可以借鉴 非常方便批量管理
> 
> wg手势触发我觉得是用过最快的，没什么拖泥带水，但是一些功能性稍缺，我现在都不知道怎么粘贴大段文字，还要带标点符号那种的，你知道怎么操作吗？？
是的  有些功能只能靠自己写,,,,,,,,,这些基础功能只能日后逐渐完善,,,对有编程功底的人比较友好,
复制文字 lua脚本:
function wgAction(pos,wid,pid,exe,title,mode)
  WG.SetClipboardText ("需要粘贴的文字")
  WG.SendKeys('^v')
end

**Comment by vr2009** [comment]
> 
我这里测试并不成功，呵呵
算了，我还是在调用Quicker动作算了

**Comment by yingDev** [comment]
按键序列就可以输出任何文字. 

不过当前版本输出abc可能引发输入法. 

下个版本增加了 "文本优先模式", 可以避免输入法干扰.

**Comment by superDMS** [comment]
> 按键序列就可以输出任何文字.
> 
> 不过当前版本输出abc可能引发输入法.
> 
> 下个版本增加了 "文本优先模式", 可以避免输入法干扰.

嗯 ,按键序列可以输出但是有些输入间隔的延迟,,,,,,还是像剪切板那样一股脑复制粘贴出来比较舒服!

**Comment by superDMS** [comment]
> > 
> 
> 我这里测试并不成功，呵呵 算了，我还是在调用Quicker动作算了

你可能把双引号去掉了  双引号要留着!

**Comment by vr2009** [comment]
> 按键序列就可以输出任何文字.
> 
> 不过当前版本输出abc可能引发输入法.
> 
> 下个版本增加了 "文本优先模式", 可以避免输入法干扰.

之前有小段的我是用按键序列的，就是觉得有些麻烦，如果有文本块直接剪贴板粘贴就比较人性化

**Comment by superDMS** [comment]
是的  按键序列输入文字不错 就是输入英文和标点符号  括号等杂七杂八容易出问题  所以我用lua剪切板来实现  笨办法

**Comment by yingDev** [comment]
有些需求开发者用不到, 所以不容易想到.
下个版本再增加个 WG.ReadText(filePath), 方便读取大段文字.

**Comment by superDMS** [comment]
> 

哇塞'?大段文字得读文件那么夸张么!! 这样灵活性就稍差了    按键序列那如果双引号引起来无论中文英文标点都能输出就好啦

**Comment by Huiduer** [comment]
对，强烈建议增加批量操作
